SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -561,43 +561,57 @@ void USpatialSender::SendAuthorityIntentUpdate(const AActor& Actor, VirtualWorke
 	if (NetDriver->StaticComponentView->HasAuthority(EntityId, SpatialConstants::ENTITY_ACL_COMPONENT_ID))
 	{
 		// Also notify the enforcer directly on the worker that sends the component update, as the update will short circuit
-		NetDriver->LoadBalanceEnforcer->QueueAclAssignmentRequest(EntityId);
+		NetDriver->LoadBalanceEnforcer->MaybeQueueAclAssignmentRequest(EntityId);
 	}
 }
 
-void USpatialSender::SetAclWriteAuthority(const Worker_EntityId EntityId, const PhysicalWorkerName& DestinationWorkerId)
+void USpatialSender::SetAclWriteAuthority(
+	const Worker_EntityId EntityId,
+	const PhysicalWorkerName& DestinationWorkerId,
+	const WorkerRequirementSet& ReadAcl,
+	const WorkerRequirementSet& ClientRequirementSet,
+	const TArray<Worker_ComponentId>& ComponentIds)
 {
 	check(NetDriver);
-	check(NetDriver->StaticComponentView->HasAuthority(EntityId, SpatialConstants::ENTITY_ACL_COMPONENT_ID));
```

Comment:
reason for deleting?

Reply:
it's unnecessary- this function will only be called if you know you have it, and the check means introducing a dependency back onto the static component view

happy to add it back if you think it is important

Reply:
nah sounds good